frontend/src/components/landing/FaqAccordion.module.scss

```diff
@@ -43,6 +43,7 @@
   }
 
   .plus-icon {
+    flex: 0 0 auto;
```

Comment:
flex-grow is 0 because the icon shouldn't take up all available space (the FAQ question should), but it shouldn't shrink below its natural size either.